services/openvpn/service/manager.go

```diff
@@ -115,12 +124,13 @@ func (m *Manager) Serve(providerID identity.Identity) (err error) {
 	m.vpnServiceConfigProvider = m.sessionConfigNegotiatorFactory(primitives, m.outboundIP, m.publicIP, m.vpnServerPort)
 
 	vpnServerConfig := m.vpnServerConfigFactory(primitives, m.vpnServerPort)
-	m.vpnServer = m.vpnServerFactory(vpnServerConfig)
+	stateChannel := make(chan openvpn.State, 10)
+	m.vpnServer = m.vpnServerFactory(vpnServerConfig, stateChannel)
 
 	// register service port to which NATProxy will forward connects attempts to
 	m.natPinger.BindServicePort(openvpn_service.ServiceType, m.vpnServerPort)
 
-	log.Info(logPrefix, "starting openvpn server on port: ", m.vpnServerPort)
+	log.Info(logPrefix, "Starting openvpn server on port: ", m.vpnServerPort)
```

Comment:
Case should be consistent, i.e. lowercase

Reply:
So logs most places we do with `log.Info("Uppercase ..")`.
Maybe You're mixing with `errors.New("lowercase ..")` which is Golang convention.

Reply:
No @Waldz, I'm talking about the logging messages. They are not consistent, thus no, in most places, we don't do capitalization of the log messages. Quite often, the error is being logged as-is, resulting in a lowercase message as well. Perhaps it's a good addition for our devbook if we can agree on the choice?